app/assets/stylesheets/govuk_publishing_components/components/_layout-header.scss

```diff
@@ -0,0 +1,34 @@
+// This component relies on styles from GOV.UK Frontend
+
+.govuk-header--production .govuk-header__container {
```

Comment:
Should this use the 'gem-' namespace?

Reply:
Good point raised here @nickcolley. I would argue that we should keep the prefix of the component when extending it with modifiers in order to identify the source of the component. It can be indeed more difficult to figure out where the modifier class comes from though.

Reply:
Since the root component still uses the upstream namespace, your modifier could show that it belongs in this gem, while still having the indication that most of the code is upstream?

Reply:
Otherwise people may find it confusing that these modifiers are not in GOV.UK Frontend.

It also can mean that if GOV.UK Frontend introduces a modifier that you do not conflict (not so likely in this case, but as a general rule it could be sensible)

Reply:
Sounds reasonable, will update.